ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -143,6 +143,13 @@
     reason="Test runs ONLY on AWS deployed cluster",
 )
 
+aws_based_platform_required = pytest.mark.skipif(
+    (
+        config.ENV_DATA["platform"].lower() != "aws"
+        and config.ENV_DATA["platform"].lower() != ROSA_PLATFORM
```

Comment:
```suggestion
        or config.ENV_DATA["platform"].lower() != ROSA_PLATFORM
```

Reply:
If the platform is not aws AND not rosa then skip TC
if either platform then the condition will be false and will not skip the TCs.

Reply:
You are right. The condition looks ok.